lib/crypto.js

```diff
@@ -535,6 +535,12 @@ ECDH.prototype.generateKeys = function generateKeys(encoding, format) {
   return this.getPublicKey(encoding, format);
 };
 
+ECDH.prototype.generatePublicKey = function generateKeys(encoding, format) {
```

Comment:
I think the function name here needs to be changed from `generateKeys` to `generatePublicKey`